features/user/impl/src/commonMain/kotlin/app/campfire/user/StoreUserRepository.kt

```diff
@@ -85,6 +86,7 @@ class StoreUserRepository(
     return userStore.stream(StoreReadRequest.cached(Unit, refresh = false))
       .filterNot { it is StoreReadResponse.Loading || it is StoreReadResponse.NoNewData }
       .map { it.requireData() }
+      .distinctUntilChanged()
```

Comment:
This was an odd bugger. Without this multiple User events would pump down to observers (its possible with store) and cause the Collections store implementation to catch in odd ways due to cancellations. making this distinctUntilChanged at the source here fixes that (no need to update observers if the object hasn't changed.